Uplift/Views/Onboarding/OnboardingView.swift

```diff
@@ -49,7 +69,14 @@ class OnboardingView: UIView {
             tableView.clipsToBounds = false
             tableView.separatorStyle = .none
             self.tableView = tableView
-            self.addSubview(tableView)
+            addSubview(tableView)
+        }
+
+        // No Classes/Gyms
+        if showingClasses && classNames?.isEmpty ?? false || !showingClasses && gymNames?.isEmpty ?? false {
```

Comment:
can you put `showingClasses && classNames?.isEmpty ?? false` and `!showingClasses && gymNames?.isEmpty ?? false` into variables so its clearer what each of these mean?